test/connectors/ODSConnectorTest.scala

```diff
@@ -16,17 +16,19 @@
 
 package connectors
 
+import models.Liability.IncomeTaxDue
+import models.ODSModels.{SaTaxpayerDetails, SelfAssessmentList}
+import models.{Amount, PensionTaxRate, TaxSummaryLiability}
+import org.mockito.Matchers.{eq => eqTo, _}
+import org.mockito.Mockito._
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.mockito.MockitoSugar
-import play.api.libs.json.JsValue
+import uk.gov.hmrc.http.{HeaderCarrier, HttpGet, HttpReads}
+import uk.gov.hmrc.play.config.ServicesConfig
 import uk.gov.hmrc.play.test.UnitSpec
-import org.mockito.Mockito._
-import org.mockito.Matchers.{eq => eqTo, _}
 import utils.TestConstants._
 
 import scala.concurrent.{ExecutionContext, Future}
-import uk.gov.hmrc.http.{HeaderCarrier, HttpGet, HttpReads}
-import uk.gov.hmrc.play.config.ServicesConfig
 
 class ODSConnectorTest extends UnitSpec with MockitoSugar with ScalaFutures {
```

Comment:
Is it possible to use WireMock here instead?

Reply:
Sure, I'll change this